### PR TITLE
editing a published translation cannot edit contributors

### LINKED ISSUE #424: Remove the ability to edit authors/translators on translations
Not consistent and is broken as of now.

### PR DESCRIPTION
**Pull request by vinyll**
fix #424

### CODE REVIEW

File: cafebabel/tests/test_translations.py [hunk 1]
```diff
@@ -326,7 +326,7 @@ def test_translation_published_should_have_translator(
              f'article title') in response)
     assert (f'<a href="/fr/profile/{translator.id}/" class=translator-link>'
             in response)
-    assert f'{translator}' in response
+    assert translator in response
```
Comment (davidbgk) [inline cafebabel/tests/test_translations.py]: Tests are failing because of this, maybe `str(translator) in response` would be enough.

Comment (vinyll) [reply cafebabel/tests/test_translations.py]: Funny that's what I just pushed ;)

Comment (davidbgk) [reply cafebabel/tests/test_translations.py]: You sure?!

Comment (vinyll) [reply cafebabel/tests/test_translations.py]: Yes but the pushed had been rejected after master update :/

**Review by davidbgk — APPROVED** [review]
Maybe indent the `if` in template now the review is done?

**Pull request merged**